### DISCUSSION TITLE
How to remove see documentation text form everywhere in orchadcore cms

### DISCUSSION THREAD

**Discussion by syedshadab19** [OrchardCMS/OrchardCore#12122] [Q&A] [answered]
Hi I wanted to remove the see documentation text form everywhere in orchadcore cms as you can see it appears in most of the the places like Body Part etc, Please find the below snapshot showcasing the see documentation text.

![image](https://user-images.githubusercontent.com/25971751/182022369-59862a53-c464-499c-b72a-8c62ebe3812b.png)

**Comment by hishamco** [comment]
It's a part of content part hint if I'm not wrong. Is there a reason for remove it?

**Reply by ns8482e** [reply]
May be white label 😅

**Reply by syedshadab19** [reply]
@hishamco its part of content part hint can you please suggest on how to remove it?

**Comment by ns8482e** [comment, marked as answer]
@syedshadab19 You need to override every shape from where you need to remove such link in your own admin theme

**Reply by syedshadab19** [reply]
@ns8482e  Can you let me know the steps on how to do that?

**Reply by ns8482e** [reply]
It’s Liquid part edit you can find it in source of Liquid module

**Reply by syedshadab19** [reply]
ok Thank You, @ns8482e
